.buildkite/fpm-pipeline.yml

```diff
@@ -1,3 +1,27 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+env:
+  SETUP_GVM_VERSION: "v0.5.1"
+  IMAGE_UBUNTU_X86_64: "family/core-ubuntu-2204"
+  DOCKER_REGISTRY: "docker.elastic.co"
+  STAGING_IMAGE: "${DOCKER_REGISTRY}/observability-ci"
+  MAKEFILE: "fpm"
+  FILES_TO_CHECK: ".buildkite/scripts/changeset/fpm"
```

Comment:
nit: Can this include >1 files? Or does it suport regex, hence, indeed, it may match >1 files? If yes, should we rename to `FILES_TO_CHECK_REGEX`?

Reply:
I have moved list of files to be checked in a separate file fpm/llvm-apple. So if any other files/directories will be necessary to check, those will be added to corresponding file. That was done, since [BK does not support regexes within the env variables](https://forum.buildkite.community/t/define-value-of-environment-variable-as-regex/1511), and fails while parsing pipeline.yml. May be the env var "FILES_TO_CHECK" name is not that accurate, I'll rename it.